plugins/acss-kit/assets/foundation/sass/_grid.scss

```
@@ -0,0 +1,101 @@
+:root {
+  --gap: 1.75rem;
+}
+
+:where([data-grid]) {
+  --gap: 1.5rem;
+  --grid-column-min-width: 20.3125rem;
+  --grid-gap: var(--gap);
+  --grid-padding: 10px;
+  --grid-justify-content: center;
+  --grid-align-content: center;
+  display: grid;
```

Stylelint (17.10.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)